packages/terra-visually-hidden-text/docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Terra Visually Hidden Text
+
+Visually Hidden Text is text designed for screen readers that should not be visible to the UI. Some components such as badge will have a visual indication to note their hierarchy on the page, but that indication is lost when a screen reader is used on the web page. Visually Hidden text would allow your application to provide that additional context to a screen reader.
```

Comment:
May add: **Here are some examples: ** and then bullet each reason you listed below instead of paragraph format